admin/javascript/SSNotice.js

```diff
@@ -0,0 +1,205 @@
+/**
+* File: SSNotice.js
+*
+* Replace the old statusMessage/errorMessage
+* with a custom library: ss.notice
+*
+* Allows for concistent notices/message display throughout the CMS
+* via simple JavaScript function calls:
+* - ss.notice.message(text, [options]);
+* - ss.notice.success(text, [options]);
+* - ss.notice.warning(text, [details], [options]);
+* - ss.notice.error(text, [details], [options]);
+**/
+(function($, root) {
+
+	// local working version of the library object
+	var lib = {};
+
+	/**
+	 * Library settings
+	 */
+	lib.settings = {
+		parentSelector: '.cms-content-fields',
+		$notice: $('<p class="message"/>'),
```

Comment:
This is just another way to create a P node with a class in jQuery. Since we are creating notices on the fly and each one is different, we can't really have them already in the template?